_includes/contribution-guide-nav.html

```diff
@@ -30,20 +30,38 @@
           </div>
         </div>
   </a>
-  <a href="/documentation/contributing/releases/">
-    {% if page.url == "/documentation/contributing/releases/" %}
+  <div class="font-other-page"><b>See Pull Requests on GitHub:</b></div>
+  <a href="https://github.com/ros-planning/moveit/pulls">
+    {% if page.url == "https://github.com/ros-planning/moveit/pulls" %}
     <div class="row no-gutters">
       <img src="/assets/install_page/current_page_left.png" class="current-page-image-left">
       <img src="/assets/install_page/current_page_right.png" class="current-page-image-right">
       <div class="font-current-page">
         {% else %}
         <div>
-          <div class="row font-other-page">
+          <div class="row font-other-page" style="text-indent: 10px">
             {% endif %}
-            Releases
+	    → MoveIt (ROS 1)
           </div>
         </div>
   </a>
+  <a href="https://github.com/ros-planning/moveit2/pulls">
+    {% if page.url == "https://github.com/ros-planning/moveit2/pulls" %}
+    <div class="row no-gutters">
+      <img src="/assets/install_page/current_page_left.png" class="current-page-image-left">
+      <img src="/assets/install_page/current_page_right.png" class="current-page-image-right">
+      <div class="font-current-page">
+        {% else %}
+        <div>
+          <div class="row font-other-page" style="text-indent: 10px">
+            {% endif %}
+	    → MoveIt (ROS 2)
```

Comment:
Why are we calling this `MoveIt (ROS 2)` instead of `MoveIt 2`? Same with MoveIt 1 above?

Reply:
"MoveIt 1" is not really a thing and "MoveIt + MoveIt 2" looked a bit strange.... What do you think?